Classes/Service/DeeplService.php

```diff
@@ -28,12 +27,11 @@ final class DeeplService implements LoggerAwareInterface
     public function __construct(
         FrontendInterface $cache,
         ClientInterface $client,
-        GlossaryRepository $glossaryRepository,
-        ProcessingInstruction $processingInstruction
+        ProcessingInstruction $processingInstruction,
```

Comment:
We could have switched all properties to private CPP, but let's do that at a later point.